src/IncludedCommands/BasicCommands/createteam.lua

```diff
@@ -13,21 +13,26 @@ local Command = BaseCommand:Extend()
 Creates the command.
 --]]
 function Command:__new()
-    self:InitializeSuper({"createteam","addteam"},"BasicCommands","Creates teams of given colors.")
+    self:InitializeSuper({"createteam","addteam"},"BasicCommands","Creates teams of given colors and given name.")
 
     self.Arguments = {
         {
             Type = "brickColors",
             Name = "Colors",
             Description = "Colors to create.",
         },
+        {
+            Type = "string",
+            Name = "Name",
+            Description = "Name to use.",
```

Comment:
Allowing a name does reduce the step of calling `renameteam`, but it breaks the existing option of just going with the color's name. Colors are sometimes filtered, which causes a lot of problems. Cmdr supports optional fields ([see `ff` for example](https://github.com/TheNexusAvenger/Nexus-Admin/blob/7ff82711b9f0e856a875b3cdee5d99ab92909df4/src/IncludedCommands/UsefulFun/ff.lua#L27)), so I would like this to be optional as well. This also changes the name line.